### PR TITLE
CASMPET-7178: LDAP service change commands fixed

### PR DESCRIPTION
**Pull request by ethanholen-hpe**
# Description

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/security_and_authentication/Change_the_LDAP_Server_IP_Address_for_Existing_LDAP_Server_Content.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Follow the steps in only one of the sections below:
     AUTH_FQDN="auth.cmn.${SYSTEM_NAME}.${SITE_DOMAIN}"
 
     function get_master_token {
-      curl -ks -d client_id=admin-cli -d username="${MASTER_USERNAME}" -d password="${MASTER_PASSWORD}" \
+      curl -ks -d client_id=admin-cli -d username="${MASTER_USERNAME}" --data-urlencode password=${MASTER_PASSWORD} \
```
Comment (kimjensen-hpe) [inline operations/security_and_authentication/Change_the_LDAP_Server_IP_Address_for_Existing_LDAP_Server_Content.md]: Do you need quotes around ${MASTER_PASSWORD} ?

**Review by kimjensen-hpe — APPROVED** [review]

**Review by mtupitsyn — APPROVED** [review]

**Review by mtupitsyn — APPROVED** [review]

**Pull request merged**